frontend/src/components/organisms/AddingPlayerForm/AddingPlayerForm.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { Button, Grid, TextField, Typography } from "@material-ui/core";
+import { VerticalSpace } from "../../atoms/Shared/VerticalSpace";
+import { useFormik } from "formik";
+import * as yup from "yup";
+import { UserProfileRestApi } from "../../../restapi/players-profiles";
+import { EntityIdGenerator } from "../../idGenerator/EntityIdGenerator";
+
+const validationSchema = yup.object({
+  name: yup.string().required("To pole jest wymagane."),
+  surname: yup.string().required("To pole jest wymagane."),
+  email: yup
+    .string()
+    .email("Niepoprawny adres e-mail.")
+    .required("To pole jest wymagane."),
+  phone: yup
+    .string()
+    .min(9, "Wprowadź przynajmniej 9 znaków.")
+    .required("To pole jest wymagane."),
+});
+
+function AddingPlayerForm() {
+  //TODO dead code below might be used for TODO below
+  // const tournamentId = useParams();
+  // console.log(tournamentId);
+  // const tourIdString: string = (Object.values(
+  //   tournamentId
+  // )[0] as string).substring(1);
+  // console.log(tourIdString);
+
+  const formik = useFormik({
+    initialValues: {
+      name: "",
+      surname: "",
+      email: "",
+      phone: "",
+    },
+    validationSchema: validationSchema,
+    onSubmit: async (values) => {
+      try {
+        await UserProfileRestApi().postPlayersProfile({
+          playerId: EntityIdGenerator.generate(),
+          firstName: values.name,
+          lastName: values.surname,
+          phoneNumber: values.phone,
+          emailAddress: values.email,
+        });
+        window.location.reload();
+        //TODO add modal window with confirmation AND add
```

Comment:
Modal? Chyba jakis tekst na tym sheet wystarczy :)

Reply:
![image](https://user-images.githubusercontent.com/31566345/113338137-fd257f00-9328-11eb-8bdf-771d3d8e9aa4.png)
no takie cudo dobrze wygląda i przyda się nie raz :-)

Reply:
A to to sie jakos inaczej nazywa :)